R/guide-colorbar.r

```diff
@@ -1,38 +1,38 @@
 #' Continuous colour bar guide
 #'
-#' Colour bar guide shows continuous color scales mapped onto values.
+#' Colour bar guide shows continuous colour scales mapped onto values.
 #' Colour bar is available with `scale_fill` and `scale_colour`.
 #' For more information, see the inspiration for this function:
 #' \href{http://www.mathworks.com/help/techdoc/ref/colorbar.html}{Matlab's colorbar function}.
 #'
 #' Guides can be specified in each `scale_*` or in [guides()].
 #' `guide="legend"` in `scale_*` is syntactic sugar for
-#' `guide=guide_legend()` (e.g. `scale_color_manual(guide = "legend")`).
+#' `guide=guide_legend()` (e.g. `scale_colour_manual(guide = "legend")`).
 #' As for how to specify the guide for each scale in more detail,
 #' see [guides()].
 #'
 #' @inheritParams guide_legend
 #' @param barwidth A numeric or a [grid::unit()] object specifying
-#'   the width of the colorbar. Default value is `legend.key.width` or
+#'   the width of the colourbar. Default value is `legend.key.width` or
 #'   `legend.key.size` in [theme()] or theme.
 #' @param barheight A numeric or a [grid::unit()] object specifying
-#'   the height of the colorbar. Default value is `legend.key.height` or
+#'   the height of the colourbar. Default value is `legend.key.height` or
 #'   `legend.key.size` in [theme()] or theme.
 #' @param frame.colour A string specifying the colour of the frame
 #'   drawn around the bar. If `NULL` (the default), no frame is drawn.
 #' @param frame.linewidth A numeric specifying the width of the frame
 #'   drawn around the bar.
 #' @param frame.linetype A numeric specifying the linetype of the frame
 #'   drawn around the bar.
-#' @param nbin A numeric specifying the number of bins for drawing colorbar. A
-#'   smoother colorbar for a larger value.
-#' @param raster A logical. If `TRUE` then the colorbar is rendered as a
-#'   raster object. If `FALSE` then the colorbar is rendered as a set of
+#' @param nbin A numeric specifying the number of bins for drawing colourbar. A
```

Comment:
Add an article before "colourbar" (If you can have more than one, then "a," if not, then "the.")